backend/infrahub/core/schema/definitions/core/__init__.py

```diff
@@ -1,5 +1,7 @@
 from typing import Any
 
+from ...generic_schema import GenericSchema
```

Comment:
I think it's fine to have a single dot to indicate a relative import from the current module, but I'm not a big fan of two or more dots.